security/s2a/examples/greeter_client/main.go

```diff
@@ -0,0 +1,96 @@
+/*
+ *
+ * Copyright 2020 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+// Package main implements a client for Greeter service.
+package main
+
+import (
+	"context"
+	"crypto/tls"
+	"crypto/x509"
+	"flag"
+	"google.golang.org/grpc/credentials"
+	"io/ioutil"
+	"log"
+	"time"
+
+	"google.golang.org/grpc"
+	pb "google.golang.org/grpc/security/s2a/examples/helloworld"
+)
+
+var (
+	serverAddr string
+	port       string
+	rootCert   string
+	certFile   string
+	keyFile    string
+)
+
+func main() {
+	serverAddr := flag.String("server address", "localhost", "address of the server")
```

Comment:
Please change the flag name to "server_address".

Reply:
Also, the flag should be the full address of the server, e.g. "localhost:50051".